### ISSUE TITLE
Time slot generator generates several wrong times in Blocks E and F

### ISSUE THREAD

**Issue by yorkshire-pudding** [backdrop-ops/events.backdropcms.org#42]
The issue started from Slot 15 when generating times for Backdrop Live October 2024

- Dates and times ran in and it did seem that then the actual time field then was wrong which also then needed correcting.
- It would make sense to ensure there is always a space
- Where they did run together then I not only had to insert a space but a number too
- The final slot (Bonus on block F) used PST rather than PDT which caused me a bit of head scratching looking at the UK times.  My guess is at one point Backdrop live spanned a change in time plans.

**Comment by jenlampton** [comment]
> Dates and times ran in and it did seem that then the actual time field then was wrong 

Yep, I just looked at the site, and you're right. It looks like someone changed some of the TIME values in the Date/Time fields on the taxonomy terms. That would have broken the tool that updates the tokens -- since the tokens are based off the times in the Date/Time fields (and the TIME never changes in those fields, they are the same for every Backdrop LIVE)

I found a handful of times that had the wrong am/pm set, and from block E onward they were all off by about 5 hours, which is a strange number!  I have just gone in and set them all back to 8/9/10 2/3/4 with the correct am/pm. 

In the code, there was an issue with PST / PDT, which made the tokens (for the slots with the correct times) appear one hour off. I am pushing a fix for that up now, but the bigger problem was the content change. 

Sorry if that caused you any frustration - I wasn't expecting anyone to edit these manually, so I never would have even thought to check that they were set correctly!
